### PR TITLE
Add loan routes, abi's, and models to swap agent

### PR DESCRIPTION
**Pull request by mattBlackDesign**
This PR implements the basics of loan routes and models for integration with Liquality Agent

### CODE REVIEW

File: src/utils/contracts.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const web3 = require('./web3')
+
+const schema = {}
+
+schema.funds = require('../abi/funds')
+schema.loans = require('../abi/loans')
+schema.sales = require('../abi/sales')
+
+export const loadObject = (type, address) => {
```
Comment (harshjv) [inline src/utils/contracts.js]: Are you using this code anywhere? And `export const` won't work with node directly, right?

File: test/common.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const config = require('./config')
+const { Client, providers } = require('@liquality/bundle')
```
Comment (harshjv) [inline test/common.js]: You can use individual packages here :)

File: src/utils/contracts.js (hunk 1 above)
Comment (mattBlackDesign) [reply src/utils/contracts.js]: Good call. Not using it yet, just setting it up for later. Thanks for pointing that out 🙂

File: test/common.js (hunk 2 above)
Comment (mattBlackDesign) [reply test/common.js]: Is there any advantage to using individual packages?

**Comment by harshjv** [comment]
@mattBlackDesign closing this PR due to inactivity. Do reopen it when it is ready!

**Pull request closed without merging**